src/App.js

```diff
@@ -69,6 +69,9 @@ function App() {
     setInfoModalIsOpen(false)
   }
 
+  const [darkMode, setDarkMode] = useLocalStorage('dark-mode', false)
+  const toggleDarkMode = () => setDarkMode(!darkMode)
```

Comment:
```suggestion
  const toggleDarkMode = () => setDarkMode(prev => !prev)
```

I _think_ this is the safer way to update state when relying on the previous value of it